source/service.go

```diff
@@ -842,12 +833,18 @@ func (sc *serviceTypes) isProcessed(serviceType v1.ServiceType) bool {
 	return !sc.enabled || sc.types[serviceType]
 }
 
-func (sc *serviceTypes) isNodeInformerRequired() bool {
-	if !sc.enabled {
+// isAllOrRequired returns true if service type filtering is disabled or if any of the provided service types are present in the filter.
+// If no options are provided, it returns true.
+func (sc *serviceTypes) isAllOrRequired(opts ...v1.ServiceType) bool {
```

Comment:
```suggestion
func (sc *serviceTypes) isRequired(opts ...v1.ServiceType) bool {
```
The fact that it goes through all looks like an implementation detail to me.
=> Wdyt of renaming it more simply ?

Reply:
fixed